common.css

```diff
@@ -8,8 +8,10 @@
   --success-color: #3ded1d;
   --error-color: #f20b07;
   --shadow-color: #0066ff;
-  --search-bar-color:rgb(221, 221, 221);
-  --dark-color:black;
+  --search-bar-color: rgb(221, 221, 221);
+  --dark-color: black;
```

Comment:
give the color code here .....

Reply:
preview link is provided above so you can go through it !!

Reply:
> give the color code here .....

I have given color code there itself and I want to use colors in rgba as I want to take opacity into considerations.